### PR TITLE
Adjustments to the dummy `jj revert` command

### PR DESCRIPTION
**Pull request by ilyagr**
Follows up on #2580, cc @Ralith 

I intend to make `jj backout` work the same way after it is renamed to `rollback`, if people on Discord agree.

# Checklist

If applicable:
- (n/a) I have updated `CHANGELOG.md`
- (n/a) I have updated the documentation (README.md, docs/, demos/)
- (n/a) I have updated the config schema (cli/src/config-schema.json)
- (maybe later?) I have added tests to cover my changes

### CODE REVIEW

File: cli/src/commands/mod.rs [hunk 1]
```diff
@@ -140,6 +140,13 @@ enum Commands {
     Workspace(workspace::WorkspaceCommands),
 }
 
+/// A dummy command that accepts any arguments
+#[derive(clap::Args, Clone, Debug)]
+struct DummyCommand {
```
Comment (yuja) [inline cli/src/commands/mod.rs:145]: nit: I'd call it `DummyArgs`.

**Review by yuja — APPROVED** [review]
Nice

File: cli/src/commands/mod.rs (hunk 1 above)
Comment (ilyagr) [reply cli/src/commands/mod.rs:145]: Thanks, good point! I think I slightly prefer `DummyCommandArgs`, for the least possible ambiguity.

**Comment by Ralith** [comment]
Nice, I hoped something like this might be possible.

**Pull request merged**